tests/schema.sql

```diff
@@ -30,7 +30,7 @@ ALTER TABLE `ak`
 
 
 CREATE TABLE `typemix` (
-   `pid` int NOT NULL,
+   `pid` int unsigned NOT NULL AUTO_INCREMENT,
```

Comment:
I would prefer adding a new column at the end of this table instead of changing a existing column. 

Additionally we need the same schema change in the pgsql schema file

Reply:
I am still trying to reproduce my issue https://github.com/staabm/phpstan-dba/issues/737... At first I thought it was somehow related to the column being the primary key, for which reason I added it here. And even though there seems to be some inconsistencies here since it is already breaking some tests, it's not failing in the way I supposed it to 🙈 I think I have to continue to try reproducing this...